03/app.js

```diff
@@ -1,3 +1,11 @@
 export default function randomNumber(min, max) {
+  if(typeof min !== 'number' || typeof max !== 'number') {
+    throw new TypeError('Arguments have to be numbers.');
+  }
```

Comment:
Zdecydowanie lepiej osobno wykonać sprawdzenie, aby można było niezależnie je przetestować.
To pozwoli docelowo poinformować użytkownika, który parametr jest niepoprawny.